cmd/ftl/cmd_config.go

```diff
@@ -60,7 +64,38 @@ type configListCmd struct {
 	Module string `optional:"" arg:"" placeholder:"MODULE" help:"List configuration only in this module."`
 }
 
-func (s *configListCmd) Run(ctx context.Context, adminClient admin.Client) error {
+func setUpAdminClient(ctx context.Context, config projectconfig.Config) (ctxOut context.Context, client admin.Client, err error) {
```

Comment:
I'm not sure what the answer to this question is, just wondering: should we move this function, or part of this function, to another file? It is config-related, and it is cmd-related, but it's not really a cmd_config-_specific_ thing conceptually. I almost wonder if we should just have an analog to `backend/controller/admin/client.go` called `backend/controller/admin/local_client.go`...

Reply:
Just kidding, that file already exists :) In that case, I suppose my question is whether some or all of this logic belongs better in there. Not urgent at all of course